pkg/util/parsing.go

```diff
@@ -10,6 +10,28 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+// ParsePorts parses the given ports string into a list of ports;
+// this includes converting port ranges into separate ports
+func ParsePorts(portsString string) ([]uint32, error) {
```

Comment:
The only place that we use this function, we immediately convert the result from a `[]uint32` into a `map[uint32]struct{}`.  Should this function just return a `map[uint32]struct{}` to begin with?

Reply:
Makes sense. Updated.